css/footer.css

```diff
@@ -0,0 +1,170 @@
+.footer {
+  text-align: center;
+  background: rgba(206, 219, 215, 1);
+}
+
+.footer-wrapper {
+  margin: 0 17%;
+  padding: 0;
+}
+
+.footer-row {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.footer-copy {
+  font-family: Poppins;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 22px;
+  color: #333333;
+  margin-top: 70px;
+  margin-bottom: 40px;
+  display: inline-block;
+}
+
+.back-to-top {
+  width: 300px;
+  height: 60px;
+  line-height: 60px;
+  text-align: center;
+  margin-top: 50px;
+  margin-bottom: 40px;
+}
+
+.back-to-top a {
+  color: #000000;
+  font-family: Poppins;
```

Comment:
@andsemenov Everything here looks great but when I run this locally I don't see this font
<img width="755" alt="Screenshot 2021-03-02 at 10 23 22" src="https://user-images.githubusercontent.com/42038541/109634850-a1a57d00-7b41-11eb-8880-d3709d709332.png">

Reply:
Thanks for this. I added link to font family, it works for me. Could you check it again, please?

Reply:
yes, all good now and looking just like the design - thank you!